test/reductions/thomasvandoren/test/TestScanCountsInherit.good

```diff
@@ -1,2 +1,2 @@
-TestScanCounts.chpl:8: warning: scan has been serialized (see note in $CHPL_HOME/STATUS)
+TestScanCountsInherit.chpl:8: warning: scan has been serialized (see issue #5760)
```

Comment:
This future now passes, at least for me.  :)

Reply:
Cool! Could you please do `git rm test/reductions/thomasvandoren/test/TestScanCountsInherit.future` to turn it into a regular passing test?

Reply:
I'll try to do that tonight.  I wasn't sure if you'd want to run the test on more platforms before removing the future.